Apps/WebClient/src/ClientApp/src/models/configData.ts

```diff
@@ -68,6 +69,10 @@ export interface WebClientConfiguration {
     minPatientAge: number;
     // Gets or sets the maximum age of a dependent
     maxDependentAge: number;
+    // Gets or sets the client IP as reported by the server
+    clientIP?: string;
```

Comment:
does not match ClientIPAddress from the server

Reply:
Changed server to match